llvm/test/CodeGen/AMDGPU/insert_waitcnt_for_all.ll

```diff
@@ -0,0 +1,199 @@
+; Testing the -amdgpu-precise-memory-op option
```

Comment:
Please generate the test using `update_llc_test_checks`, much easier to update if/when things change.

Also I think you don't need `-verify-machineinstrs`. It's expensive and runs anyway when EXPENSIVE_CHECKS is on.